packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1240,8 +1240,11 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			}
 		}
 
-		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-		async function onKeyDown(event: any) {
+		async function onKeyDown(event: KeyboardEvent) {
+			// Although the onChangeHandler also marks the editor content as changed, it's important
+			// to set changed to true as soon as possible. See https://github.com/laurent22/joplin/issues/8960.
+			props.setChanged(true);
```

Comment:
What even are actually processed by this handler? Would it be triggered by non-printable keys too? Because in that case the note wouldn't actually be changed